otter/test/rest/test_groups.py

```diff
@@ -134,6 +135,37 @@ def test_format_state_dict_with_convergence(self, config_value):
         self.assertEqual(result['desiredCapacity'], 3)
         self.assertEqual(result['pendingCapacity'], 0)
 
+    @mock.patch('otter.rest.groups.config_value')
+    def test_format_state_different_status(self, config_value):
+        """
+        When a group's status is something other than ACTIVE, it's reflected in
+        the output.
+        """
+        config_value.side_effect = {'convergence-tenants': ['11111']}.get
+        active = {
+            '1': {'name': 'n1', 'links': ['links1'], 'created': 't'},
+            '2': {'name': 'n2', 'links': ['links2'], 'created': 't'},
+            '3': {'name': 'n3', 'links': ['links3'], 'created': 't'}}
+        state = GroupState(
+            '11111',
+            'one',
+            'test',
+            active,
+            {},  # Ignored!
+            None,
+            {},
+            True,
+            ScalingGroupStatus.ERROR,
+            desired=10)
+        result = format_state_dict(state)
+        self.assertEqual(result['desiredCapacity'], 10)
+        self.assertEqual(result['pendingCapacity'], 7)
+
+        # And a non-convergence tenant still gets old-style data
```

Comment:
Is the point of this part that a non-convergence tenant also gets the error status?  Should we also test that the convergence tenant gets the status too?